### PR TITLE
Add match priorities to AML settings API

### PR DESCRIPTION
**Pull request by mikerebilly**
## Summary
Add match priorities to AML settings.

I know we have a camelCase rule for property names, but in this case I'd prefer to keep them named the way they are named for confidence (very-strong, strong, etc.) and source types (pep, adverse-media, etc)

If we have to enforce this it will add a bit of complexity to the use of the API for both confidence and source types as they're both kebab-case enum values elsewhere in the schema. (Admittedly not much extra complexity.)

There's no real breaking issue with that, but it's some sort of cognitive load to remember that the camelCase are mapped 1:1 with the respective kebab-case enum values for whatever use case the API consumer has for it.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by mikerebilly** [comment]
I posited a discussion in the description. Will update accordingly after a final decision is made, although I am suspecting which direction this will go.

**Comment by ashkarpetin** [comment]
I think we should use camelCase, see https://github.com/Rebilly/api-definitions/pull/778

**Comment by mikerebilly** [comment]
@Rebilly/php @Rebilly/js @Rebilly/docs please review, camelCase and styling fixed

**Review by ashkarpetin — APPROVED** [review]

**Review by cesarlevel — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Review by tamasvigh — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by noon-dawg — COMMENTED** [review]
This is looking great!

I added two very minor suggestions

File: openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+type: object
+description: Priority settings.
```
[the remark this answers is not in the document]
Comment (mikerebilly) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:2]: I have a follow-up PR where I have to add `priority` to the AmlCheck response schema, so I'll apply these suggestions then.

[the remark this answers is not in the document]
Comment (mikerebilly) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:2]: Added here #1387

[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:2]: Looks good!

**Pull request merged**